src/fragments/lib-v1/project-setup/android/use-existing-resources/use-existing-resources.mdx

```diff
@@ -8,7 +8,7 @@ Before you can add an existing AWS resource to an Android application, the appli
 
 First, locate your project’s `res` folder. For example, if the name of your project is *MyAmplifyApp*, you can find the `res` folder at the following location, `MyAmplifyApp/app/src/main/res`:
 
-![GSA](/images/project-setup/2_useExistingResources.png)
+![Shows where the res folder is located in your android project](/images/project-setup/2_useExistingResources.png)
```

Comment:
I think this could be more descriptive since the image is circling the location of the folder within the project explorer window. I think it's good that we have the path already mentioned, so maybe we could leverage that. 

One possible change to this could be, "Screenshot of the res folder in Android Studio's project explorer, from the path mentioned above"